### DISCUSSION TITLE
This example has already been tested for build a TLS connection successfully?

### DISCUSSION THREAD

**Discussion by tigerBeh12** [nsc/SwiftTLS#17] [Q&A]
As the title say.

**Comment by nsc** [comment]
Yes. As mentioned in the README, there is a server running on https://swifttls.org and you can connect to any server running TLS 1.2 or 1.3 using the command line tool:
swift run -c release tls client --connect server.example.org

**Reply by tigerBeh12** [reply]
You're really good. I have a general look at the code today, and it's too difficult for me to implement such a low-level detailed implementation. 5555555
